src/agent/commands/results.publish.ts

```diff
@@ -45,7 +48,10 @@ export class ResultsPublishCommand implements cm.IAsyncCommand {
             releaseEnvironmentUri: this.executionContext.variables["release.environmentUri"],
             releaseUri: this.executionContext.variables["release.releaseUri"],
             platform: platform,
-            config: config
+            config: config,
+			runTitle: runTitle,
+			fileNumber: fileNumber,
```

Comment:
Can we avoid the filenumber parameter and send it along with runtitle? we can avoid duplicating the logic in the readers.